src/applications/disability-benefits/all-claims/tests/pages/vaMedicalRecords.unit.spec.jsx

```diff
@@ -271,7 +271,7 @@ describe('VA Medical Records', () => {
                 diabetesmelitus: true,
               },
               treatmentDateRange: {
-                from: '2010-04-XX',
+                from: '2010-04-15',
```

Comment:
Here as well, `day` value will always be stored as 'XX' as there is no `day` field input. Applies to entirety of changes to this file.